support-frontend/app/controllers/DigitalSubscription.scala

```diff
@@ -59,8 +62,13 @@ class DigitalSubscription(
     val productPrices = priceSummaryServiceProvider.forUser(false).getPrices(DigitalPack, promoCode)
 
     Ok(views.html.main(
-      title, id, js, css, description, canonicalLink, hrefLangLinks, productPrices = Some(productPrices)
-    )).withSettingsSurrogateKey
+      title, id, js, css, description, canonicalLink, hrefLangLinks
+    ) {
+      val priceLines = productPrices.map(p =>
+        s"window.guardian.productPrices = ${outputJson(p)}"
```

Comment:
I prefer to avoid using single letter names in maps, up to you but I'd try and give it a more descriptive name

Reply:
good idea, although this line is actually moved from the main.scala.html so I didn't change that